src/interpreter/utils.ts

```diff
@@ -125,7 +125,7 @@ export function formatIssueContext(issue: ValidationIssue): string {
       return `Wrong Profile Name: expected "${issue.context.expected}", but got "${issue.context.actual}"`;
 
     case 'wrongProfileVersion':
-      return `Wrong Profile Version: expected "${composeVersion(
+      return `Version is not matching: expected "${composeVersion(
```

Comment:
How about present tense?

```suggestion
      return `Version does not match: expected "${composeVersion(
```